js/apps/admin-ui/CONTRIBUTING.md

```diff
@@ -46,31 +46,20 @@ Every time you create a commit it should be automatically linted and formatted f
 pnpm lint
 ```
 
-## Integration testing with Cypress
+## Integration testing with Playwright
 
-This repository contains integration tests developed with the [Cypress framework](https://www.cypress.io/).
+This repository contains integration tests developed with the [Playwright framework](https://playwright.dev/).
 
 ### Prerequisites
 
 Ensure the Keycloak and development server are running as [outlined previously](#running-the-keycloak-server) in this document.
 
 ### Running the tests
 
-You can run the tests using the interactive graphical user interface using the following command:
-
-```bash
-pnpm cy:open
-```
+You can run the tests using the interactive graphical user Visual Studio Code:
```

Comment:
There are no instructions here to actually do this in VSCode.

Reply:
it's really simple just press the button...

Reply:
ahh I thought it was standard in VS code, but it's not, made it a link to the extension

Reply:
Looks good, resolving.